packages/amplify-cli/src/extensions/amplify-helpers/get-all-envs.ts

```diff
@@ -3,15 +3,7 @@ import { stateManager } from 'amplify-cli-core';
 /**
  * Get all locally configured environments
  */
-export const getAllEnvs = (): string[] => {
-  let allEnvs: string[] = [];
-
-  const envInfo = stateManager.getTeamProviderInfo(undefined, {
-    throwIfNotExist: false,
-    default: {},
-  });
-
-  allEnvs = Object.keys(envInfo);
-
-  return allEnvs;
-};
+export const getAllEnvs = (): string[] => Object.keys(stateManager.getLocalAWSInfo(undefined, {
```

Comment:
Does this provide all list of envs or the current one ?

Reply:
It's a list of all the environments that have been pulled locally